tests/neo-vm.Tests/Converters/ScriptConverter.cs

```diff
@@ -34,7 +35,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             {
                 case JsonToken.String:
                     {
-                        if (reader.Value is string str) return str.FromHexString();
+                        if (reader.Value is string str)
+                        {
+                            Assert.IsTrue(str.StartsWith("0x"), "'0x' prefix required");
```

Comment:
I don't know if this works as intended. I can see the continuous integration build passing while there are still many places where this condition is not satisfied. e.g.

https://github.com/neo-project/neo-vm/blob/c6910e929eedf4a391a88a7788c7fd5dfd9cb5e1/tests/neo-vm.Tests/Tests/OpCodes/Arrays/SETITEM.json#L233

https://github.com/neo-project/neo-vm/blob/c6910e929eedf4a391a88a7788c7fd5dfd9cb5e1/tests/neo-vm.Tests/Tests/OpCodes/Splice/CAT.json#L286

https://github.com/neo-project/neo-vm/blob/c6910e929eedf4a391a88a7788c7fd5dfd9cb5e1/tests/neo-vm.Tests/Tests/OpCodes/Splice/CAT.json#L294

https://github.com/neo-project/neo-vm/blob/c6910e929eedf4a391a88a7788c7fd5dfd9cb5e1/tests/neo-vm.Tests/Tests/OpCodes/Splice/CAT.json#L316

Reply:
I will check this

Reply:
Please check it again